src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --main-blue-color:#00acdc;
+}
+
+img {
+  padding-left: 50px;
+  align-self: center;
+}
+
+header {
+  display: flex;
+  text-transform: uppercase;
+}
+
+nav {
+  display: flex;
+  margin-left: auto;
+  align-items: center;
+  margin-right: 50px;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+a {
```

Comment:
style by classes instead by tag.
add classes for img, header, nav, ul, a tags....